### ISSUE TITLE
dotnet new --help display usage message twice

### ISSUE THREAD

**Issue by yizhang82** [dotnet/cli#9041]
## Steps to reproduce

1. dotnet new --help
2. Look at the help message

## Expected  behavior

Usage message only shown once

## Actual behavior

```
Usage: new [options]

Options:
  -h, --help          Displays help for this command.
  -l, --list          Lists templates containing the specified name. If no name is specified, lists all templates.
  -n, --name          The name for the output being created. If no name is specified, the name of the current directory is used.
  -o, --output        Location to place the generated output.
  -i, --install       Installs a source or a template pack.
  -u, --uninstall     Uninstalls a source or a template pack.
  --type              Filters templates based on available types. Predefined values are "project", "item" or "other".
  --force             Forces content to be generated even if it would change existing files.
  -lang, --language   Specifies the language of the template to create.


Usage: new [options]

Options:
  -h, --help          Displays help for this command.
  -l, --list          Lists templates containing the specified name. If no name is specified, lists all templates.
  -n, --name          The name for the output being created. If no name is specified, the name of the current directory is used.
  -o, --output        Location to place the generated output.
  -i, --install       Installs a source or a template pack.
  -u, --uninstall     Uninstalls a source or a template pack.
  --type              Filters templates based on available types. Predefined values are "project", "item" or "other".
  --force             Forces content to be generated even if it would change existing files.
  -lang, --language   Specifies the language of the template to create.
```
## Environment data
`dotnet --info` output:

.NET Command Line Tools (2.1.103)

Product Information:
 Version:            2.1.103
 Commit SHA-1 hash:  60218cecb5

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.103\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.6
  Build    : 74b1c703813c8910df5b96f304b0f2b78cdf194d

**Comment by nguerrera** [comment]
This is fixed in 2.13xx, but does repro with 2.1.1xx and 2.1.2xx.

cc @mlorbetske @livarcocc Do you know which change fixed this? I'm guessing we wouldn't patch 2.1.1xx or 2.1.2xx for this, right?

**Comment by livarcocc** [comment]
I agree. I find it hard that this meets the bar for patching. The fix in 300 seems all right.

**Comment by mlorbetske** [comment]
This would be addressed by #9029 and #9030, these changes didn't flow to 2.1.1xx and 2.1.2xx via product construction.
